site/js/modules/webui/assignment.js

```diff
@@ -563,6 +563,11 @@ function handlerProxyRegrade(path, params, context, container) {
 function proxyRegrade(params, context, container, inputParams) {
     let course = context.courses[params[Routing.PARAM_COURSE]];
     let assignment = course.assignments[params[Routing.PARAM_ASSIGNMENT]];
+    console.log(
+            course.id, assignment.id,
+            inputParams.dryRun, inputParams.overwrite,
+            inputParams.cutoff, inputParams.users, inputParams.wait,
+    );
```

Comment:
This log seem off.
Was it accidentally left in?
If not, it would need to be debug (and the payload should probably wrapped in an object).